spec/features/appointments_spec.rb

```diff
@@ -578,13 +578,17 @@
   end
 
   describe 'fulfilment' do
+    before do
+      place = create :place, organization: @user.organization
+      @agenda = create :agenda, place: place
+    end
     it 'controls are only displayed for passed appointments' do
       convict = create :convict
       create :areas_convicts_mapping, convict: convict, area: @user.organization.departments.first
 
       apt_type = create :appointment_type, :with_notification_types
       slot = create :slot, date: Date.civil(2025, 4, 14), starting_time: Time.now - 1.minutes,
-                           appointment_type: apt_type
+                           appointment_type: apt_type, agenda: @agenda
```

Comment:
the slot use the agenda place and we need it to be equal to the user organization